code/matt/python/lab17.py

```diff
@@ -0,0 +1,119 @@
+import json
+
+
+class ContactList:
+    def __init__(self):
+        self.contacts = []
+
+    def load(self):
+        # 1) open 'contacts.json' with option 'r' for read
+
+        with open("contacts.json", "r") as f:
+            # 2) get the text from the file
+            contacts_content = f.read()
+            # 3) convert the text into a python dictionary (json.loads)
+        contacts_dict = json.loads(contacts_content)
+        # 4) get the list of contacts out of the dictionary
+        contacts_dict["contacts"]
+        # 5) assign the list of dictionaries to self.contacts
+        self.contacts = contacts_dict["contacts"]
+
+    def count(self):
+        # return the length of self.contacts
+        return len(self.contacts)
+        ...
+
+    def save(self):
+        # 1) open 'contacts.json' with open 'w' for write
+        with open("contacts.json", "w") as f:
```

Comment:
Not a requirement for this assignment but what happens if this file doesn't exist?